gensim/sklearn_api/basemodel.py

```diff
@@ -11,7 +11,7 @@
 from abc import ABCMeta, abstractmethod
 
 
-class BaseSklearnWrapper(object):
+class BaseTransformer(object):
```

Comment:
Right, it's always asked, but not always supported. The scikit-learn convention for not supporting incremental training is to simply not implement `partial_fit()`. Why not adopt the same convention for scikit-learn compatible classes here?